snappy.cc

```diff
@@ -252,6 +279,42 @@ inline char* IncrementalCopy(const char* src, char* op, char* const op_limit,
       if (SNAPPY_PREDICT_TRUE(op >= op_limit)) return op_limit;
     }
     return IncrementalCopySlow(src, op, op_limit);
+
+#elif SNAPPY_HAVE_PWR9
```

Comment:
could/should `#ifdef defined(_ARCH_PWR9)` be used directly?

Reply:
I am just copying what was done already for SSE,  the same could be done via SSE preprocessor flags yet it uses it's own.  I am simply trying to keep the style as I found it.

Reply:
Would say that he normalized version is used correclty (`SNAPPY_HAVE_VSX`).

@dmiller423 , remember to flag this as "solved"